backend/services/core/src/types/author.ts

```diff
@@ -0,0 +1,23 @@
+import { UUID } from '~/src/shared';
+
+/**
+ * A custom author represents a group or entity of which some members can post as.
+ */
+export interface CustomAuthor {
+  id: UUID,
+  name: string,
+  name_en?: string,
+  image_url?: string,
+  created_at: Date,
+  updated_at: Date,
+}
+
+export interface Author {
```

Comment:
If the database is changed to enforce data consistency, then this type should be adjusted too (if type 'Member' then `mandate_id` and `custom_id` is `never` and so on).

Reply:
This depends on what we decide on for the previous comment above regarding database consistency and if we should keep the type column etc. But yes, this type can be more enforced.

Reply:
Might not be worth the effort, the backend type safety overall is questionable.

Reply:
Knex did not like this at all. Keeping as is, now we have database enforced safety with checks